### ISSUE TITLE
is imklog being tested?

### ISSUE THREAD

**Issue by rgerhards** [rsyslog/rsyslog#3071]
This needs to be investigated and possibly a test be added.
Note: this needs to be done as root account. As such, adding a test makes sense only after we have setup containers in CI which support that (and even then it is questionable, as the container probably cannot access the host's kernel log - to be investigated as well).

**Comment by rgerhards** [comment]
actually, there exists test `queue-encryption-da.sh` - so this issues boils down to finding a way to actually test this under CI runs (the broader "root issue").

**Comment by rgerhards** [comment]
not worthj to add in 2026 if not already done, so closing.
